java/connector-node/README.md

```diff
@@ -34,6 +34,7 @@ Sometimes, you need to specify the shared library path. For example, when progra
 
 ```
 RISINGWAVE_ROOT=$(git rev-parse --show-toplevel)
+JAVA_ROOT=$RISINGWAVE_ROOT/java
 CONNECTOR_ROOT=$JAVA_ROOT/connector-node
 # Build shared library file
 cd $RISINGWAVE_ROOT && cargo build -p risingwave_java_binding
```

Comment:
In the latest main, we don't need to explicit build the rust java_binding. Could you help to update this doc?

Reply:
Yes, we don't need it. In https://github.com/risingwavelabs/risingwave/pull/10229, the shared library will be built and added to the jar during maven build, and the shared library can be loaded from the java, so the we don't need to set the `java.library.path` now.